backend/main.py

```
@@ -160,19 +253,30 @@ async def root():
         "status": "active"
     }
 
-
 @app.get("/api/scenarios/")
 async def get_scenarios(
-    current_user: User = Depends(get_current_user),
+    current_user: Optional[User] = Depends(get_current_user_optional),
```

Ruff (0.13.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)